File: oklog3-example/src/main/java/com/github/simonpercic/oklog3example/data/api/model/response/show/ShowResponse.java
package com.github.simonpercic.oklog3example.data.api.model.response.show;

/**
 * @author Simon Percic <a href="https://github.com/simonpercic">https://github.com/simonpercic</a>
 */
public class ShowResponse {

    private long id;

    private String name;

    private int runtime;

    private ShowNetworkResponse network;

    // region getters

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public int getRuntime() {
        return runtime;
    }

    public ShowNetworkResponse getNetwork() {
        return network;
    }

    // endregion getters
}

File: example-kotlin/src/main/kotlin/com/github/simonpercic/oklogexamplekotlin/data/api/model/response/show/ShowResponse.kt
package com.github.simonpercic.oklogexamplekotlin.data.api.model.response.show

/**
 * @author Simon Percic [https://github.com/simonpercic](https://github.com/simonpercic)
 */
class ShowResponse {

    var id: Long = 0
    var name: String? = null
    var runtime: Int = 0
    var network: ShowNetworkResponse? = null
}
